CODE_OF_CONDUCT.md

```diff
@@ -1,3 +1,340 @@
-# Code of Conduct
+# Conda Organization Code of Conduct
```

Comment:
Hm, do we actually want this replicated verbatim? Can we just include the short version and include a link to the full version?

Reply:
A fair point. I was concerned about redundancy issues. That's a better solution.